src/python/tests/gff3/test_simplifier.py

```diff
@@ -456,37 +456,37 @@ def test_gffsimplifier_with_genome(
             "mirna/gene.gff",
             "mirna/gene_simped.gff",
             does_not_raise(),
-            id="gene + primary_transcript + miRNA",
+            id="ncRNA_gene + miRNA_primary_transcript + miRNA",
         ),
         param(
             "mirna/pseudogene.gff",
             "mirna/pseudogene_simped.gff",
             does_not_raise(),
-            id="gene + primary_transcript - miRNA",
+            id="ncRNA_gene + miRNA_primary_transcript - miRNA",
         ),
         param(
             "mirna/nogene.gff",
             "mirna/nogene_simped.gff",
             does_not_raise(),
-            id="primary_transcript + miRNA",
+            id="miRNA_primary_transcript + miRNA",
         ),
         param(
             "mirna/pseudo_nogene.gff",
             "mirna/pseudo_nogene_simped.gff",
             does_not_raise(),
-            id="primary_transcript - miRNA",
+            id="miRNA_primary_transcript - miRNA",
         ),
         param(
             "mirna/unsupported_tr.gff",
             "",
             raises(GFFParserError, match="Unknown subtype"),
-            id="gene + primary_transcript + mRNA, not supported",
+            id="ncRNA_gene + miRNA_primary_transcript + mRNA, not supported",
         ),
         param(
             "mirna/two_primary.gff",
             "",
             raises(GFFParserError, match="too many sub_features"),
-            id="gene + 2x primary_transcript, not supported",
+            id="gene + 2x miRNA_primary_transcript, not supported",
         ),
```

Comment:
Here I would keep the old names, because they correspond to the input biotypes, not the results (the input files have not changed)